_component_design/tooltip.md

```diff
@@ -13,28 +13,304 @@ last-modified: 2017-08-17
+- **Data visualization:** Use this trigger variation to present robust data sets aggregated into a data visualization.
+
+
+{% endcolumn %}
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Examples of tooltips generated from various trigger types."] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/help/tooltips/tooltip-triggers.svg){:width="100%"}
+
 {% endfigure %}
 {% endcolumn %}
 </div>
+
+## Text conventions
+{: .hxSectionTitle}
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+{: .hxSubSectionTitle}
+
+- To keep the tooltip text concise, you should use short sentences and sentence fragments. 
+- Try to keep your tooltips under three lines of text including a link if you need one.
+
+
+{% endcolumn %}
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Most tooltips should be very short and only provide enough information for the user to better understand the context. Tooltips can also be used to reveal information that is truncated or hidden by default."] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/help/tooltips/tooltip-text-conventions-1.svg){:width="100%"}
+
+{% endfigure %}
+{% endcolumn %}
+</div>
+
+## Tooltip variations
+{: .hxSectionTitle}
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Help text with a link
+{: .hxSubSectionTitle}
+
+This variation is used when a tooltip need to provide access to a singular external webpage.
+
+- This link should always be an external link and should open in a new tab. 
+- Be sure you always include an external link icon
+- If you need additional inputs or interactions on a tool tip, e.g. a dropdown or selector, use the larger popover pattern.
+
+See [Help links]({{site.baseurl}}/components/help-link.html) for link specifications.
+
+
+{% endcolumn %}
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Spacing for Tooltip with an associated link"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/help/tooltips/tooltip-text-with-link.svg){:width="100%"}
+
+{% endfigure %}
+{% endcolumn %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Large tooltips
+{: .hxSubSectionTitle}
+
+Occassionally a tooltip needs to display a robust set of data that falls outside the bounds of traditional text conventions. The larger tooltips still follow the text conventions and max-width: 400px guidelines assosiated with tooltips and should be used sparingly. While the occasional outlier is ok, if you are consistently using the large tooltip pattern it may be an indication of less than optimal design decisions.
```

Comment:
Occasionally, a tooltip * ...

"The larger tooltips follow the same text conventions and sizing guidelines." Remove "400px guidelines assosiated with tooltips and should be used sparingly."

Reply:
Remove "While the occasional outlier is ok, if you are consistently using the large tooltip pattern it may be an indication of less than optimal design decisions."